src/swati/A15_ArrayExample.java

```diff
@@ -0,0 +1,97 @@
+package swati;
+
+public class A15_ArrayExample {
+
+	int[] arr = { 15, 21, 18, 31, 12 };
+
+	// Maximum
+	int maxArray() {
+		int max = 0;
+		for (int i = 0; i < arr.length - 1; i++) {
```

Comment:
If you are taking the 0th index as max number then you can start the loop from index 1 as well, same for other methods